### ISSUE TITLE
New bug that just started happening today (CustomTube 1.3.0 Beta 1)

### ISSUE THREAD

**Issue by gameshowrecaps** [lightbeam24/CustomTube#191]
![image](https://github.com/lightbeam24/CustomTube/assets/57842610/9304c83b-476f-47de-b904-2b98690e291f)

Possible new A/B testing I got involved in?

**Comment by lightbeam24** [comment]
Yeah that happens sometimes. For some reason Polymer decides to not provide `visitorData` sometimes, which causes this. I don't know why it does that though. I do have an idea of how to possibly fix this, but I haven't yet tried.
